autogpt_platform/backend/backend/executor/manager.py

```diff
@@ -569,6 +570,9 @@ def _on_node_execution(
                     f"Failed node execution {node_exec.node_exec_id}: {e}"
                 )
 
+            if stats is not None:
+                stats["error"] = e
```

Comment:
Request: can we make sure the error being persisted here in the db (`AgentNodeExecution` table) is just a message, not a full stacktrace ?

Reply:
if it's a full stacktrace we should exclude it from the stats dict when persisting.

Reply:
Why not the whole thing sorry?

Reply:
We should avoid persisting long text in our row-based db at all cost, plus our ORM will keep loading this multiple times throughout the node execution.

Reply:
okay its chill i belive